src/components/tags/PopularTags.tsx

```diff
@@ -1,26 +1,36 @@
-import React, { FC } from 'react';
+import React, { FC, useState, useEffect } from 'react';
 import classNames from 'classnames';
 import style from './tags.module.scss';
-
-const tagList: string[] = [
-  'welcome',
-  'implementations',
-  'codebaseShow',
-  'introduction',
-];
+import BASE_URL from '../../utils/baseUrl';
 
 const onFilterTags = () => {};
 
 const PopularTags: FC = () => {
+  const [tags, setTags] = useState<string[]>([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch(`${BASE_URL}tags`);
```

Comment:
It would be better to use something like `$BASE_URL}/tags` I think. Or create enum for these routes and just use it.